cpp/include/cudf/utilities/span.hpp

```diff
@@ -123,12 +123,14 @@ struct host_span : public span_base<T, Extent, host_span<T, Extent>> {
 
   constexpr host_span() noexcept : base() {}  // required to compile on centos
 
-  template <typename C, std::enable_if<is_host_span_supported_container<C>::value>* = nullptr>
+  template <typename C,
+            typename std::enable_if_t<is_host_span_supported_container<C>::value>* = nullptr>
```

Comment:
For any `std::***_t` traits construct, you don't need the `typename` prefix. 

```suggestion
            std::enable_if_t<is_host_span_supported_container<C>::value>* = nullptr>
```

Reply:
thought I removed this :expressionless: